Fix translation file names

Atticus29 opened the pull request:
# Description

Closes #2959 . Error was:

> Got error 'PHP message: PHP Warning: include_once(content/lang/misc/aboutproject.en.php): Failed to open stream: No such file or directory in /var/www/html/ecdysis/portal/misc/contacts.php on line 4; PHP message: PHP Warning: include_once(): Failed opening 'content/lang/misc/aboutproject.en.php' for inclusion (include_path='.:/usr/share/pear:/var/www/html/ecdysis/portal:/var/www/html/ecdysis/portal/config/:/var/www/html/ecdysis/portal/classes/') in /var/www/html/ecdysis/portal/misc/contacts.php on line 4'

I think that was in Ecdysis.

I found myself confused why there was a translation template file. That seemed like an older pattern? Correct me if I'm mistaken.

# Pull Request Checklist:

# Pre-Approval

- [ ] There is a description section in the pull request that details what the proposed changes do. It can be very brief if need be, but it ought to exist.
- [ ] Hotfixes should be branched off of the `master` branch and PR'd using the **merge** option (not squashed) into the `hotfix` branch.
- [ ] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [ ] All new text is preferably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages), and the [spreadsheet tracking internationalizations](https://docs.google.com/spreadsheets/d/133fps9w2pUCEjUA6IGCcQotk7dn9KvepMXJ2IWUZsE8/edit?usp=sharing) has been updated either with a new row or with checkmarks to existing rows.
- [ ] There are no linter errors
- [ ] New features have responsive design (i.e., look aesthetically pleasing both full screen and with small or mobile screens)
- [ ] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed
- [ ] If any files have been reformatted (e.g., by an autoformatter), the reformat is its own, separate commit in the PR
- [ ] Comment which GitHub issue(s), if any does this PR address
- [ ] If this PR makes any changes that would require additional configuration of any Symbiota portals outside of the files tracked in this repository, make sure that those changes are detailed in [this document](https://docs.google.com/document/d/1T7xbXEf2bjjm-PMrlXpUBa69aTMAIROPXVqJqa2ow_I/edit?usp=sharing).

# Post-Approval

- [ ] It is the code author's responsibility to merge their own pull request after it has been approved
- [ ] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [ ] If this PR represents a merge into the `hotfix` branch, remember to use the **merge** option (i.e., no squash).
- [ ] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [ ] If this PR represents a merge from the `hotfix` branch into the `master` branch use the **squash & merge** option
  - [ ] a subsequent PR from `master` into `Development` should be made with the **merge** option (i.e., no squash).
  - [ ] **Immediately** delete the `hotfix` branch and create a new `hotfix` branch
  - [ ] increment the Symbiota version number in the symbase.php file and commit to the `hotfix` branch.
- [ ] If the dev team has agreed that this PR represents the last PR going into the `Development` branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place. Follow the release protocol [here](https://github.com/BioKIC/Symbiota/blob/master/docs/release-protocol.md).
- [ ] Don't forget to delete your feature branch upon merge. Ignore this step as required.

Thanks for contributing and keeping it clean!

Atticus29 committed:
convert aboutproject.en_template to aboutproject.en and add the current pattern of language file handling to the template php files that reference it

themerekat commented:
@Atticus29 , I think we've moved all the things that are likely to need per-portal adjustment to the "templates" directory within "content/lang". Should we consider moving aboutproject there too?

themerekat reviewed (changes requested):
See previous question. Not a dealbreaker, but could be nice to streamline

Atticus29 committed:
move aboutproject.foo to content/lang/templates and update references to aboutproject.foo.php with new path

Atticus29 commented:
> @Atticus29 , I think we've moved all the things that are likely to need per-portal adjustment to the "templates" directory within "content/lang". Should we consider moving aboutproject there too?

Fixed in 02fcbb0b0.
